src/Makefile_Ecore.am

```diff
@@ -85,6 +86,8 @@ lib/ecore/efl_io_file.c \
 lib/ecore/efl_io_copier.c \
 lib/ecore/efl_io_buffered_stream.c \
 lib/ecore/efl_promise.c \
+lib/ecore/efl_promise2.c \
+lib/ecore/efl_promise2_priv.c \
```

Comment:
I did this split so efl_promise2.c does not get too big, but no problem I will merge both files